tests/Block/FeatureMediaBlockServiceTest.php

```diff
@@ -30,6 +30,7 @@ protected function setUp(): void
         $this->blockService = new FeatureMediaBlockService(
             $this->twig,
             new Pool('default'),
+            /* @phpstan-ignore-next-line */
```

Comment:
Why do you need to ignore it?

Reply:
Because phpstan complains about the missing generic information. AFAIK this is not possible for phpunit mocks.

Reply:
I dunno if it's different when doing 
```
$this->admin = $this->createStub(AdminInterface::class);
```
when admin is a property with a `@var` phpdoc.

The other option would be to do 
```
/** @var ... */
$admin = $this->createStub(AdminInterface::class);
```

Reply:
To me it looks better with the var thing instead of ignoring the line, which seems like a todo or a place to come later and fix something

Reply:
I agree